Sources/OpenAI/Client.swift

```diff
@@ -827,6 +830,40 @@ public final class Client {
                 }
             }
     }
+    
+    // MARK: Content Filter
+    /**
+     The filter aims to detect generated text that could be sensitive or unsafe coming from the API. It's currently in beta mode and has three ways of classifying text- as safe `0`, sensitive `1`, unsafe `2`, or error `-1`. The filter will make mistakes and we have currently built it to err on the side of caution, thus, resulting in higher false positives.
+
+     - Parameters:
+       - prompt: The prompt(s) to generate content filter request for.
+       - completion: A closure to be called once the request finishes.
+                     The closure takes a single argument, an integer, indicating the safety of the prompt.
+
+     - SeeAlso: https://beta.openai.com/docs/engines/content-filter
+     
+     - Note: Content filter requests are free, and should cost 0 tokens.
+     */
+
+    public func contentFilter(prompt: String, completion: @escaping (Int) -> Void) {
```

Comment:
What do you think about creating an enumeration type to encapsulate this? For example:

```swift
public enum Safety: Int {
    case safe = 0
    case sensitive = 1
    case unsafe = 2
}
```

(with conformance to `CustomStringConvertible`, `Equatable`, `Comparable`, `Hashable`, and `ExpressibleByIntegerLiteral` for completeness)

Reply:
Fun! I think this is a great idea. I'll take a look into it right now.